apps/src/templates/rubrics/productTourHelpers.js

```diff
@@ -1,41 +1,62 @@
+import React from 'react';
+
 import i18n from '@cdo/locale';
 import evidenceDemo from '@cdo/static/ai-evidence-demo.gif';
 
+// intro.js-react allows a string or a react component for the intro prop.
+// Providing a string that was written by a translator is risky, because it
+// could contain malicious HTML. This helper method wraps the string in a react
+// component, which will take care of sanitizing the string.
+const sanitize = unsafe => <span>{unsafe}</span>;
```

Comment:
I'm not 100% sure if this will work, but using a react fragment instead of a span might trigger React's virtual DOM to parse the unsafe without having to add a dummy element, which should lighten the DOM

```suggestion
const sanitize = unsafe => <>{unsafe}</>;
```

Reply:
Fragments seem to work! I verified that the product tour still renders correctly, and that this approach still escapes malicious HTML.